app/etc/di.xml

```diff
@@ -407,11 +407,16 @@
     </type>
     <type name="Magento\Framework\Interception\Config\Config">
         <arguments>
-            <argument name="cache" xsi:type="object">Magento\Framework\App\Cache\Type\Config</argument>
             <argument name="reader" xsi:type="object">Magento\Framework\ObjectManager\Config\Reader\Dom\Proxy</argument>
             <argument name="cacheId" xsi:type="string">interception</argument>
         </arguments>
     </type>
+    <type name="Magento\Framework\Interception\Config\CacheManager">
+        <arguments>
+            <argument name="cache" xsi:type="object">Magento\Framework\App\Cache\Type\Config</argument>
+            <argument name="configWriter" xsi:type="object">Magento\Framework\App\ObjectManager\ConfigWriter\Filesystem</argument>
```

Comment:
Why did you introduce an explicit type here instead of adding a preference for `\Magento\Framework\App\ObjectManager\ConfigWriterInterface`?